CHANGELOG.md

```diff
@@ -2,6 +2,27 @@
 
 All notable changes to this project will be documented in this file.
 
+## [6.0.1](https://github.com/voxpupuli/puppetboard/tree/6.0.1) (2025-02-13)
```

Comment:
I just noticed that previews releases were prefixed with v, we should keep that?

Reply:
urgs, yes, please, will update, mom